### PR TITLE
Sync: Maestro

### PR DESCRIPTION
**Pull request by malmstein**
Task/Issue URL: https://app.asana.com/0/1201493110486074/1205182950038768/f

### Description
Critical path tests for Sync

### Steps to test this PR

Run maestro tests for sync: 
`maestro test .maestro/sync_flows`

### CODE REVIEW

**Comment by malmstein** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #3297** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/3297" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/duckduckgo/Android/3297?utm_source=stack-comment).

File: .maestro/sync_flows/1_-_create_account.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+---
+- launchApp:
+    clearState: true
+    stopApp: true
+- runFlow: "open_sync.yaml"
+- tapOn:
+      id: "com.duckduckgo.mobile.android.debug:id/trailingSwitch"
+- tapOn: "Turn On Sync"
+- tapOn: "Not Now"
+- tapOn: "Not Now"
```
Comment (jpbochi) [inline .maestro/sync_flows/1_-_create_account.yaml:11]: don't you need add some `assertVisible` in between each `tapOn: "Not Now"`? I'd guess that there's a risk of double tapping the button of one step, intead of moving two steps.

**Comment by CDRussell** [comment]
@malmstein, on my emulator i see it failing because it gets stalled at this permission screen. Can you add (optional) step to allow permission?

Screenshot 2023-08-11 at 12 56 44

**Review by CDRussell — APPROVED** [review]

**Comment by malmstein** [comment]
Thanks @CDRussell !

**Pull request merged**